pkg/runners/zero/program.go

```diff
@@ -20,12 +20,7 @@ type Program struct {
 	Builtins []sn.Builtin
 }
 
-func LoadCairoZeroProgram(content []byte) (*Program, error) {
-	cairoZeroJson, err := zero.ZeroProgramFromJSON(content)
-	if err != nil {
-		return nil, err
-	}
-
+func LoadCairoZeroProgram(cairoZeroJson *zero.ZeroProgram) (*Program, error) {
```

Comment:
Ah, I like this change.
I would want to use this new API in my PR as well because I need to have a "raw" json object in addition to the runnerzero-processed `program`.